nknd.go

```diff
@@ -396,22 +396,41 @@ func GetID(seeds []string, publickey []byte) ([]byte, error) {
 		return id, nil
 	}
 
-	log.Infof("get ID from localnode faild, no ID in ledger: %v", err)
+	if err != nil {
+		log.Errorf("get ID from local ledger error: %v", err)
+	} else {
+		log.Infof("get no ID from local ledger")
+	}
```

Comment:
How about `log.Infof("get ID from localnode return %v with err %v", id, err)` directly?

Reply:
We shouldn't log as info if there is an error. The default log level will soon be error